processor/redactionprocessor/go.mod

```diff
@@ -0,0 +1,40 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor
+
+go 1.17
+
+require (
+	github.com/stretchr/testify v1.7.0
+	go.opencensus.io v0.23.0
+	go.opentelemetry.io/collector v0.37.0
+	go.opentelemetry.io/collector/model v0.37.0
```

Comment:
Again, sorry for the delay. Please update the version to 0.38.0

Reply:
Updated for 0.39.0